test/Makefile.include

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2013-2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+EXTRA_DIST += test/functional test/util/bctest.py test/util/bitcoin-util-test.py test/util/data
+
+check-local:
+	@echo "Running test/util/bitcoin-util-test.py..."
+	$(PYTHON) $(top_builddir)/test/util/bitcoin-util-test.py
+	@echo "Running test/functional/test_runner.py..."
+	$(top_builddir)/test/functional/test_runner.py nodehandling httpbasics invalidblockrequest blockchain rpcnamedargs.py
```

Comment:
I like this! But please make a var for it so that we can have obvious diffs when we add tests. Something like:
```
FUNCTIONAL_TESTS =
FUNCTIONAL_TESTS += nodehandling
FUNCTIONAL_TESTS += httpbasics
...
$(top_builddir)/test/functional/test_runner.py $(FUNCTIONAL_TESTS)
```

Going even further, as a later step, we may be able to use test_runner.py as an automake test launcher, so that each test gets parallelized as its own make job.